### PR TITLE
Move ModT function declaration to SimpleFunctionMultiArgsTabularOverl…

### PR DESCRIPTION
**Pull request by LucGenetier**
…oadImplementations

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Mod.txt [hunk 1]
```diff
@@ -237,4 +237,4 @@ Error(Table({Kind:ErrorKind.InvalidArgument},{Kind:ErrorKind.InvalidArgument}))
 Table({Result:0},{Result:1})
 
 >> Mod(3, [-1, 0, 1, 2, 3, Blank()])
-Table({Result:0},{Result:Error({Kind:ErrorKind.Div0})},{Result:0},{Result:1},{Result:0},{Result:Error({Kind:ErrorKind.Div0})})
+Table({Result:0},{Result:Error({Kind:ErrorKind.Div0})},{Result:0},{Result:1},{Result:0},{Result:Error({Kind:ErrorKind.InvalidArgument})})
```
[the remark this answers is not in the document]
Comment (LucGenetier) [reply src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Mod.txt]: Fixed

**Review by CarlosFigueiraMSFT — APPROVED** [review]
:shipit:

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**